tests/Earthfile

```diff
@@ -1075,17 +1075,11 @@ command:
 command-explicit-global:
     DO +RUN_EARTHLY --earthfile=command-explicit-global.earth
 
-command-nested-global:
-    DO +RUN_EARTHLY --earthfile=command-nested-global.earth --target="+test-overriding --foo=expected"
-
 function:
     RUN echo "hello function" >./message.txt
     DO +RUN_EARTHLY --earthfile=function.earth --target=+all-positive
     DO +RUN_EARTHLY --earthfile=function.earth --should_fail=true --target=+test-command-fails
 
-function-explicit-global:
-    DO +RUN_EARTHLY --earthfile=function-explicit-global.earth
```

Comment:
Note this test is no longer needed as it's covered by https://github.com/earthly/earthly/commit/97e8876af2bfe5464d66e79c5defce2bdb7a7ece